app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_listed = command.split(" ")
+    source_file = command_listed[1]
```

Comment:
Consider using more descriptive variable names. For example, `source_file` could be `source_file_path` to indicate it's a path.